Sources/Messenger/Activation/Activation.swift

```diff
@@ -0,0 +1,81 @@
+//
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Header comments should be consistent with project patterns.](https://app.codacy.com/app/FabrizioBrancati/SwiftyBot/pullRequest?prid=3796373)